docs/cloud/alerts-notifications/add-pagerduty-notification-configuration.md

```diff
@@ -0,0 +1,59 @@
+<!--
+title: "Add pagerduty notification configuration"
+sidebar_label: "Add pagerduty notification configuration"
+custom_edit_url: "https://github.com/netdata/netdata/blob/master/docs/cloud/alerts-notifications/add-pagerduty-notification-configuration.md"
+sidebar_position: "1"
+learn_status: "Published"
+learn_topic_type: "Tasks"
+learn_rel_path: "Operations/Alerts"
+learn_docs_purpose: "Instructions on how to add notification configuration for pagerduty"
+-->
+
+From the Cloud interface, you can manage your space's notification settings and from these you can add specific configuration to get notifications delivered on pagerduty.
```

Comment:
Same comments as on Discord. 

- pagerduty -> Pagerduty
- slack -> Slack
- and the number bullets don't seem to work? Everything says 1

Reply:
@shyamvalsan will check:
* pagerduty -> Pagerduty, this in fact will place as PagerDuty
* slack -> Slack

for the number bullets won't do anything since 1. are correctly converted to actual running list elements, saw an issue or PagerDuty due to indentation missing will fix through that. great catch!
![image](https://user-images.githubusercontent.com/82235632/215066506-afde64db-449d-4ebb-a206-2a2dcb4e8984.png)